src/views/tickets/tickets.jsx

```diff
@@ -139,7 +140,7 @@ export class Tickets extends Component {
   }];
 
   getTickets = (context) => {
-    axios.get(`/api/tickets`, { headers: { "Authorization": `Bearer ${context.user.accessJwt}` } })
+    axios.get(`/api/tickets`, axios.get(`/api/tickets`, makeAuthHeaders(context)))
```

Comment:
Looks like an extra axios call got in here somehow and is breaking the page load occasionally